dev-tools/packaging/packages.yml

```diff
@@ -98,10 +98,10 @@ shared:
         mode: 0600
         config: true
       /etc/{{.BeatName}}/beats/filebeat-{{ beat_version }}-{{.GOOS}}-{{.AgentArchName}}.tar.gz:
```

Comment:
can you cherry pick also a commit where snapshot is used also here on this line?

Reply:
https://github.com/elastic/beats/pull/17077/files

This one right?

Reply:
yes please

Reply:
I have included #17077 #17368 #17016 #16898 in this PR can you take another look?